db/migrate/20210105131930_remove_funding_organisation_fields.rb

```diff
@@ -0,0 +1,7 @@
+class RemoveFundingOrganisationFields < ActiveRecord::Migration[6.0]
+  def change
+    remove_column :activities, :funding_organisation_name, :string
```

Comment:
Might be completely unnecessary, but rolling this back should probably fill in those fields with BEIS details rather than leaving them blank.